client/extensions/woocommerce/app/settings/payments/payments-location-currency.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+class SettingsPaymentsLocationCurrency extends Component {
+
+	static propTypes = {
+		className: PropTypes.string,
```

Comment:
For each section/card - you don't need to pass in className if you don't want - just the outermost container needs it (e.g. for `woocommerce` )